### ISSUE TITLE
Issue with grid and offsets

### ISSUE THREAD

**Issue by Pautomagi** [zurb/foundation-sites#7612]
I cant get the offsets to work with a project I am doing.

I did some testing, and it turns out if I don’t import the foundation-grid and rather create a class specifying 
```
   @include grid-column-offset(6);
   @include grid-column-size(2);
```
That works. But not doing ex. large-offset-10 in the markup.

I just installed a new clean foundation project trying to use the offset classes, and still not working.

Can someone please tell me that I am doing something wrong, or confirm this bug ?

**Comment by gakimball** [comment]
Can you post an HTML example where you use a large offset class and it doesn't work?

**Comment by nska02** [comment]
I'm facing the same issue (I guess).

Here is a simple example:
```html
<div class="row" style="border: black 1px solid;">
    <h1>ROW 1</h1>
    {% for i in 1..6 %}<div class="small-2 columns" style="border: black 1px dashed;"><p class="text-center">C{{i}}</p></div>{% endfor %}
</div>

<div class="row" style="border: black 1px solid;">
    <h1>ROW 2</h1>
    <div class="small-2 columns" style="border: black 1px dashed;"><p class="text-center">C1</p></div>
    <div class="small-2 small-offset-1 columns" style="border: black 1px dashed;"><p class="text-center">C2 offset1</p></div>
    <div class="small-2 columns" style="border: black 1px dashed;"><p class="text-center">C3</p></div>
</div>
```

Results:
<img width="1210" alt="screen shot 2015-12-22 at 19 32 51" src="https://cloud.githubusercontent.com/assets/11373782/11962787/d5c04cd2-a8e2-11e5-90f0-fe6057409f7c.png">

I would expect C2 (row2) to be 1 column offset (ie. in "vertical alignement" with row 1 c3).

**Comment by Pautomagi** [comment]
I experience this with my own markup and even with a fresh install of foundation using the example from the documentations.

**Comment by nska02** [comment]
Update: same code works if use the foundation (compiled) **css** file (instead of **scss** version).
Any idea, why?

<img width="1210" alt="screen shot 2015-12-22 at 22 07 40" src="https://cloud.githubusercontent.com/assets/11373782/11965925/7342e400-a8f8-11e5-9728-6a150c37d65e.png">

For information, I experienced same pattern issue (not working with scss, working with css) between the 6.0.5 and 6.0.6 update on top-bar floating/width states (I already referenced this issue in the respective commit).

**Comment by DominoCMS** [comment]
Grid "offset" and "uncentered" not working in Foundation 6.0.0 < 6.1.1 in the scss version.

To fix uncentered problem add float: left; to grid-column-unposition

@mixin grid-column-unposition {
  position: static;
  margin-left: 0;
  margin-right: 0;
  float: left; // add this to work
}

**Comment by gakimball** [comment]
Fixed uncentered via 34cb85622e459cae13aea90252a54fe8a18dc81d
